example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -354,7 +354,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```

Comment:
similarly, wouldn't have expected the ios min version to change in the example, but not critical to the changes here

Reply:
i always forget to undo these two pieces, 👎